GitTfs/Core/GitTfsRemote.cs

```diff
@@ -542,17 +546,17 @@ private string ProcessChangeset(ITfsChangeset changeset, LogEntry log)
             return commitSha;
         }
 
-        private IEnumerable<string> TranslateWorkItems(IEnumerable<string> workItemsOriginal)
+        private IEnumerable<IExportWorkItem> TranslateWorkItems(IEnumerable<IExportWorkItem> workItemsOriginal)
         {
             if (ExportWorkitemsMapping.Count == 0)
                 return workItemsOriginal;
-            List<string> workItemsTranslated = new List<string>();
+            List<IExportWorkItem> workItemsTranslated = new List<IExportWorkItem>();
             if (workItemsOriginal == null)
                 return workItemsTranslated;
             foreach (var oldWorkItemId in workItemsOriginal)
             {
-                string translatedWorkItemId = null;
-                if (oldWorkItemId != null && !ExportWorkitemsMapping.TryGetValue(oldWorkItemId, out translatedWorkItemId))
+                IExportWorkItem translatedWorkItemId = null;
+                if (oldWorkItemId != null && !ExportWorkitemsMapping.TryGetValue(oldWorkItemId.Id.ToString(), out translatedWorkItemId))
```

Comment:
`oldWorkItemId.Id.ToString()` is doubly redundant ("id" shows up twice, and the Id property is already a string). Please rename the variable to `oldWorkItem` and drop the `.ToString()`.